bin/init

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This script will ensure Portus' database is ready to be used. It will keep
+# waiting for the db to be usable, but the script will exit with an error
+# after a certain amount of failed attempts.
+#
+# The script will automatically import all the SSL certificates from
+# `/certificates` into the final system. This is needed to talk with the
+# registry API when this one is protected by TLS.
+#
+# Finally the script will start apache running Portus via mod_rails.
+
+set -e
+# check that BUNDLER_BIN is set
+if [ ! -z $BUNDLER_BIN ]; then
+    echo "BUNDLER_BIN var is not set. Please set that and try again" >&2
+    exit -1
+fi
+
+setup_database() {
+  set +e
```

Comment:
(i think) it should be enough to have that statement once (as you already have above)